programs/install/Install.cpp

```diff
@@ -886,6 +887,27 @@ namespace
                 fmt::print("Sent kill signal.\n", pid);
             else
                 throwFromErrno("Cannot send kill signal", ErrorCodes::SYSTEM_ERROR);
+
+            /// Wait for the process (100 seconds).
+            constexpr size_t num_kill_check_tries = 1000;
+            constexpr size_t kill_check_delay_ms = 100;
+            for (size_t i = 0; i < num_kill_check_tries; ++i)
+            {
+                fmt::print("Waiting for server to be killed\n");
+                if (!isRunning(pid_file))
+                {
+                    fmt::print("Server exited\n");
+                    break;
+                }
+                sleepForMilliseconds(kill_check_delay_ms);
+            }
+
+            if (isRunning(pid_file))
+            {
+                throw Exception(ErrorCodes::CANNOT_KILL,
+                    "The server process still exists after %zu ms",
```

Comment:
1. %zu does not work for `fmt`, use just `{}`
2. We have two variables, not one.

Reply:
@alexey-milovidov indeed, thanks, nice catch!
Can you please apply the [patch](https://github.com/azat/ClickHouse/commit/3aba1d98c1a203bd9be2274838a897beb9095666) manually (I don't want to spend CI time on this simple patch).